internal/services/dataprotection/data_protection_backup_instance_disk_resource_test.go

```diff
@@ -212,3 +226,21 @@ resource "azurerm_data_protection_backup_instance_disk" "test" {
 }
 `, template, data.RandomInteger)
 }
+
+func (r DataProtectionBackupInstanceDiskResource) snapshotSubscriptionId(data acceptance.TestData) string {
```

Comment:
Can we test the case when snapshot in another subscription?

Reply:
added